### PR TITLE
Add Docklands Light Railway & Porto Metro 

### PR DESCRIPTION
**Pull request by RedAuburn**
fixes https://github.com/organicmaps/organicmaps/issues/6083
part of https://github.com/organicmaps/organicmaps/issues/3419

adds:
* Docklands Light Railway (DLR)
* Porto Metro

image
Screenshot from 2023-09-25 23-20-12
I don't want to mess about with my map generator config to build an MWM for Porto, but it should work the same as the DLR.

Added partially as a request from my friend, who works on the DLR 😄

### CODE REVIEW

**Review by pastk — APPROVED** [review]

**Comment by RedAuburn** [comment]
Modified to make subway_entrance-london generic, as they display as a London Underground entrance otherwise (see https://github.com/organicmaps/organicmaps/issues/6145)

before


after

**Comment by RedAuburn** [comment]
(forgot to rebase)

File: iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings [hunk 1]
```diff
@@ -2003,15 +2003,15 @@
 /* These translations are used for all type.highway.*.bridge. */
 "type.highway.footway.bridge" = "Puente";
 
-"type.highway.footway.demanding_alpine_hiking" = "Camino";
+"type.highway.footway.demanding_alpine_hiking" = "Path";
```
Comment (pastk) [inline iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings:2006]: translations are replaced again!

Comment (RedAuburn) [reply iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings:2006]: aaaaargh! damn it.

**Comment by RedAuburn** [comment]
@pastk fixed es-mx strings

File: iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings (hunk 1 above)
Comment (biodranik) [reply iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings]: What is your system locale? Which ruby version are you using?

Comment (RedAuburn) [reply iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings]: locale is en-GB, Ruby version is 3.2.2

Comment (biodranik) [reply iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings]: It looks like a bug in our patched twine. Can you try to log/debug it?

Comment (RedAuburn) [reply iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings]: i'll try to 👍

**Review by pastk — APPROVED** [review]
@vng

**Review by vng — APPROVED** [review]

**Review by biodranik — APPROVED** [review]
LGTM to merge after the rebase.

**Comment by RedAuburn** [comment]
rebased :)

**Comment by biodranik** [comment]
Thanks!

**Pull request merged**